features/features.go

```diff
@@ -36,6 +36,13 @@ func AllFeatureSets() map[ClusterProfileName]map[configv1.FeatureSet]*FeatureGat
 var (
 	allFeatureGates = map[ClusterProfileName]map[configv1.FeatureSet]*FeatureGateEnabledDisabled{}
 
+	FeatureGateConsolePluginCSP = newFeatureGate("ConsolePluginCSP").
+					reportProblemsToJiraComponent("Management Console").
+					contactPerson("jhadvig").
+					productScope(ocpSpecific).
+					enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
+					mustRegister()
```

Comment:
You'll need to add an enhancementPR link for your feature
```suggestion
					enhancementPR("").
					mustRegister()
```